### DISCUSSION TITLE
Getting writegood working in org filetype

### DISCUSSION THREAD

**Discussion by physicophilic** [w0rp/ale#4037] [Q&A - Ask for help with problems]
Hi

Firstly, thanks to the authors for integrating ALE with writegood. It's an awesome tool. Now onto the problem:

In my vimrc I've got

```vim
let g:ale_linters = {
  \ 'markdown': ['writegood'],
  \ 'latex': ['writegood'],
  \ 'org': ['writegood'],
  \   'apkbuild': [],
  \   'vim': [],
  \   'csh': [],
  \   'elixir': [],
  \   'go': [],
  \   'hack': [],
  \   'help': [],
  \   'inko': [],
  \   'perl': [],
  \   'perl6': [],
  \   'python': [],
  \   'rust': [],
  \   'spec': [],
  \   'svelte': [],
  \   'text': [],
  \   'vue': [],
  \   'zsh': [],
  \   'v': [],
  \}

```

which I think means that I'm asking `org` filetype to use `writegood` linter. But when I do `:ALEInfo` in `org` buffer, I see no linter available; nothing enabled. 

What am I doing wrong?

**Comment by hsanson** [comment]
Unfortunatelly "org" is not available for write-good. Check `:h ale-supported-list` for the list of languages and supported tools for each of them.  Also when running :ALEInfo it shows the list of *available* tools. Only these can be enabled for the specific file type.

Feel free to add an issue to include write-good to the list of available tools for org and if up for the task create a PR to include it. Looking at how write-good support is implemented in ALE this should be easy to do. Simply create a file named `ale_linters/org/writegood.vim` that contains one line:

```
call ale#handlers#writegood#DefineLinter('org')
```

Update any tests if necessary:

  - test/linter/test_write_good.vader
  - test/handler/test_write_good_handler.vader

Update the documentation:

  -  ale.txt
  - supported-tools.md
  - ale-supported-languages-and-tools.txt
  - doc/ale-org.txt   (new file)

You can check Asciidoc implementation of writegood for examples:

  - doc/ale-asciidoc.txt
  - ale_linters/asciidoc/writegood.vim
